assignment-2/power.md

```diff
@@ -1,3 +1,127 @@
+
+
+######Clienst & Contractors
+The clients were according to [4, page 6]: the Department of Health and Human Services (HHS), Centers for Medicare & Medicaid Services (CMS), 36 States, 300 private insurers, U.S. Chief Technology Office, GAO, Media, Citizens, Social Security Administration, the Internal Revenue Service, Veterans Administration, Office of Personnel Management, Peace Corps, etc.
+
+The construction of HealthCare.gov was overseen by Centers for Medicare and Medicaid Services (CMS) which is a part of the US Department of Health and Human Services (HHS). The primary contractor was CGI who had a contract of around 250 million dollars. Next to that there were 16 official subcontractors, but the total number of subcontractors was actually 55. All under the supervison of CMS. There were also 300 private insurers. All in all there were around 4000 plans.
+
+
+
+### Who was in control?
+
+######CMS
+CMS (Centers for Medicare and Medicaid Services) was responsible for the overall coordination and planning of the project. But it did fail according to reports of the office of inspector general.
+
+
+According to one report [1] there were inadequacies in contract planning and procurement. The report states: <i>“When awarding the Federal Marketplace contracts, CMS did not always meet contracting requirements. For example, CMS did not develop an overarching acquisition strategy for the Federal Marketplace or perform all required oversight activities. Moreover, for a project of this size and importance, CMS missed opportunities to leverage all available acquisition planning tools and contracting approaches to identify and mitigate risks. Specifically, CMS did not exercise the option to plan for a lead systems integrator to coordinate all contractors’ efforts prior to the launch of the Federal Marketplace. The complexity of the Federal Marketplace underscored the need for CMS to select the most qualified contractors. However, CMS did not perform thorough reviews of contractor past performance when awarding two key contracts. CMS also made contracting decisions that may have limited the number of acceptable proposals for much of the key Federal Marketplace work. In addition, CMS selected contract types that placed the risk of cost increases for this work solely on the Government.“ </i>. 
+
+According to another report of the office of inspector general [3] CMS did not alway manage and oversee the contractor performance. The main problem in this case is the lack of control on the contract deliverables.
```

Comment:
You have a point, but I not going to elaborate. I can write another two pages easily on power, but I don't know when it is too little, enough or to much. I'm curious with the feedback of Robert Deckers coming Monday.